v2.1/src/components/lenker.tsx

```diff
@@ -8,10 +8,9 @@ import {useEnhetContextvalueState, useFnrContextvalueState} from "../hooks/use-c
 
 function Lenke(props: { href: string; children: string; target?: string; }) {
     /* eslint-disable jsx-a11y/anchor-has-content */
-    const rel = props.target ? 'noopener noreferrer' : undefined;
     return (
         <li>
-            <a {...props} className="typo-normal dekorator__menylenke" rel={rel}/>
```

Comment:
Så slik jeg leser det så sendte vi tidligere data når `rel = undefined`?
Det høres bare litt rart ut, synes jeg 🤔 

[The rel attribute specifies the relationship between the current document and the linked document. Only used if the href attribute is present.](https://www.w3schools.com/tags/att_a_rel.asp)